general/administration/connectivity.md

```diff
@@ -18,6 +18,26 @@ If you can access the server locally but not outside of your LAN, then you likel
 
 The easiest way to check for issues is by checking the logs, which can be accessed through the settings on the web frontend or in the log directory on your server. If there are no logs at all relating to web traffic, even over a LAN connection, then the server hasn't been reached at all yet. This would indicate either an incorrect address or an issue somewhere else on the network.
 
+## Debug Mode
+
+To enable debugging for Jellyfin, it is currently required to be manually enabled. Go to Jellyfin-directory/config, in the `logging.json` file, change:
+
+`"MinimumLevel": "Information",`
+
+to
+
+`"MinimumLevel": "Debug",`
+
+Jellyfin 10.4.1 and above will auto-reload the new configuration. The debug messages show up in the log with the tag DBG.
+
+## DLNA
+
+DLNA is based on uPnP. DLNA will send a broadcast signal from Jellyfin. This broadcast is limited to Jellyfin's current subnet. If using Docker Host Mode is required, otherwise the broadcast signal will only be sent in the bridged network inside of docker. 
```

Comment:
```suggestion
DLNA is based on uPnP. DLNA will send a broadcast signal from Jellyfin. This broadcast is limited to Jellyfin's current subnet. If using Docker, network in Host Mode is required, otherwise the broadcast signal will only be sent in the bridged network inside of docker. 
```